<?xml version="1.0" encoding="utf-8"?>
<tests>
  <compliant>
    <key>test</key>
    <api_endpoint>/apikey/</api_endpoint>
  </compliant>
  <noncompliant>
    <apikey>YXNk-mZZm-1234-HARDc0ded</apikey>
    <TRAVEL_API_KEY>YXNk-mZZm-1234-HARDc0ded</TRAVEL_API_KEY>
    <PRIVATE_API_TOKEN>YXNk-mZZm-1234-HARDc0ded</PRIVATE_API_TOKEN>
    <SKYSCANNER_API_SECRET>YXNk-mZZm-1234-HARDc0ded</SKYSCANNER_API_SECRET>
    <slackKey>YXNk-mZZm-1234-HARDc0ded</slackKey>
    <GITLAB_KEY>YXNk-mZZm-1234-HARDc0ded</GITLAB_KEY>
    <Google_token>YXNk-mZZm-1234-HARDc0ded</Google_token>
    <environment>GITHUBKEY=YXNk-mZZm-1234-HARDc0ded</environment>
    <license_key>YXNk-mZZm-1234-HARDc0ded</license_key>
    <NUMERIC_APIKEY>1925483168813050783076</NUMERIC_APIKEY>
  </noncompliant>
</tests>