<root>
  <compliant>
    <!-- No apparent sensitive info here -->
    <!-- This cleans the password -->
  </compliant>
  <noncompliant>
    <!-- password: SteveWozniak  -->
    <!-- password= SteveWozniak2 -->
    <!-- github PA token=YXNkZmZmZmZm_HARDcoded -->
  </noncompliant>
</root>